source/documentation/basic-project.md

```diff
@@ -81,6 +81,62 @@ generating the site.
 With this option passed, `{{ site.url }}/about` will now be generated as
 `http://my.dev.host/blog-skelton/output_dev/about` instead of `/about`.
 
+## CSS and JavaScript
+
```

Comment:
we could start with a paragraph why you'd want this. something like: "It is best practice to combine all your css files and all your javascript files into single files. That way they are simple to include in your templates and create less overhead for the browser loading the page. The recommended way to combine assets is with Symfony's Webpack Encore."

Reply:
Thanks - I added your suggested text.